src/Host.cpp

```diff
@@ -774,6 +774,10 @@ bool Host::killTimer(const QString& name)
 {
     return mTimerUnit.killTimer(name);
 }
+int Host::remainingTime(const QString& name)
```

Comment:
Can we have a blank line above this one to space it from the previous method/function?